templates/Web/Pages/Angular.Grid/src/app/app-shell/AngularGrid/grid.component.ts

```diff
@@ -1,17 +1,17 @@
 ﻿import { Component, OnInit } from '@angular/core';
-import { CONSTANTS } from '../../../constants';
-import { GridService, IGridTextItem } from './grid.service';
 import { DomSanitizer } from '@angular/platform-browser';
 import { MatIconRegistry } from '@angular/material/icon';
 
+import { GridService, IGridTextItem } from './grid.service';
+
 @Component({
   selector: 'app-grid',
   templateUrl: './grid.component.html',
   styleUrls: ['./grid.component.css']
 })
 export class GridComponent implements OnInit {
 
-  WarningMessageText = CONSTANTS.ERROR_MESSAGE.GRID_GET;
+  WarningMessageText = 'Request to get grid text failed:';
```

Comment:
I don't know if I am missing something but why is it hard coded?

Reply:
I believe it's because 

> Don't use constants for error messages and just hard code the messages in their components (giving the fact that the rest of your text and description are in plain English).

The best way to handle it would be localization (https://angular.io/guide/i18n), but as we're not doing this, I agree that constants don't bring any value because these strings are not being reused.